kalite/coachreports/static/js/coachreports/coach_reports/views.js

```diff
@@ -495,10 +130,11 @@ var GroupSelectView = Backbone.View.extend({
     This fetches group data for facilities and displays them in a drop down
     */
 
-    template: HB.template('coach_nav/group-select'),
+    template: require("./hbtemplates/group-select.handlebars"),
```

Comment:
Yes - it's because handlebars works by virtue of a plugin rather than being natively supported.